web/src/engine/websites/LittleGarden.ts

```diff
@@ -0,0 +1,78 @@
+import { Tags } from '../Tags';
+import icon from './LittleGarden.webp';
+import { Chapter, DecoratableMangaScraper, type Manga, Page } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchGraphQL, FetchRequest } from '../FetchProvider';
+
+type APIChapters = {
+    chapters: {
+        id: number,
+        name: string,
+        number: number
+    }[]
+}
+
+type ChapterIdentifier = {
+    id: number,
+    number: number
+}
+
+type APIPages = {
+    chapter: {
+        pages: {
+            original: string,
+            colored: string
+        }[]
+    }
+}
+
+@Common.MangaCSS(/https?:\/\/littlexgarden\.com\/[^/]/, 'h2.super-title')
+@Common.MangasSinglePageCSS('/mangas', 'div.listing a.no-select', Common.AnchorInfoExtractor(true))
+@Common.ImageAjax()
+
+export default class extends DecoratableMangaScraper {
+
+    public constructor() {
+        super('littlegarden', `Little Garden`, 'https://littlexgarden.com', Tags.Language.English, Tags.Language.French, Tags.Media.Manga, Tags.Media.Manhwa, Tags.Source.Scanlator);
+    }
+
+    public override get Icon() {
+        return icon;
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const slug = manga.Identifier.split('/').pop();
+        const operationName = 'chapters';
+        const query = 'query chapters($slug: String, $limit: Float, $skip: Float, $order: Float!, $isAdmin: Boolean!) {\n  chapters(limit: $limit, skip: $skip, where: {deleted: false, published: $isAdmin, manga: {slug: $slug, published: $isAdmin, deleted: false}}, order: [{field: "number", order: $order}]) {\n    published\n    likes\n    id\n    number\n    thumb\n    manga {\n      id\n      name\n      slug\n      __typename\n    }\n    __typename\n  }\n}\n';
```

Comment:
Query is hard to read, recommendation to enclose in backticks and format correctly with line-breaks and indentation

Reply:
See also other queries with escape characters

Reply:
There are also some fields which probably can be removed as well